worlds/raft/items.json

```diff
@@ -7,335 +7,460 @@
         "progression": false,
-        "name": "Head light"
+        "name": "Flippers"
     },
     {
         "id": 47007,
         "progression": false,
-        "name": "Oxygen bottle"
+        "name": "Head light"
     },
     {
         "id": 47008,
-        "progression": true,
-        "name": "Zipline tool"
+        "progression": false,
+        "name": "Advanced head light"
     },
     {
         "id": 47009,
-        "progression": true,
-        "name": "Empty bottle"
+        "progression": false,
+        "name": "Oxygen bottle"
     },
     {
         "id": 47010,
-        "progression": false,
-        "name": "Clay bowl"
+        "progression": true,
+        "name": "Zipline tool"
     },
     {
         "id": 47011,
         "progression": false,
-        "name": "Bucket"
+        "name": "Electric zipline tool"
     },
     {
         "id": 47012,
         "progression": false,
-        "name": "Healing salve"
+        "name": "Empty bottle"
     },
     {
         "id": 47013,
         "progression": false,
-        "name": "Good healing salve"
+        "name": "Empty canteen"
     },
     {
         "id": 47014,
         "progression": false,
-        "name": "Cookingpot"
+        "name": "Bucket"
     },
     {
         "id": 47015,
         "progression": false,
-        "name": "Advanced grill"
+        "name": "Clay bowl"
     },
     {
         "id": 47016,
         "progression": false,
-        "name": "Advanced purifier"
+        "name": "Drinking glass"
     },
     {
         "id": 47017,
         "progression": false,
-        "name": "Electric purifier"
+        "name": "Healing salve"
     },
     {
         "id": 47018,
         "progression": false,
-        "name": "Medium crop plot"
+        "name": "Good healing salve"
     },
     {
         "id": 47019,
         "progression": false,
-        "name": "Large crop plot"
+        "name": "Cookingpot"
     },
     {
         "id": 47020,
-        "progression": true,
-        "name": "Grass plot"
+        "progression": false,
+        "name": "Juicer"
     },
     {
         "id": 47021,
         "progression": false,
-        "name": "Scarecrow"
+        "name": "Advanced grill"
     },
     {
         "id": 47022,
         "progression": false,
-        "name": "Sprinkler"
+        "name": "Electric grill"
     },
     {
         "id": 47023,
         "progression": false,
-        "name": "Honey"
+        "name": "Advanced purifier"
     },
     {
         "id": 47024,
+        "progression": false,
+        "name": "Electric purifier"
+    },
+    {
+        "id": 47025,
+        "progression": false,
+        "name": "Advanced small crop plot"
+    },
+    {
+        "id": 47026,
+        "progression": false,
+        "name": "Advanced medium crop plot"
+    },
+    {
+        "id": 47027,
+        "progression": false,
+        "name": "Advanced large crop plot"
+    },
+    {
+        "id": 47028,
+        "progression": false,
```

Comment:
Some items have incorrect progression flags, which is causing the unit test failure. I sent a PR directly to your repository to change some items which has fixed the unit test failure but should be double checked

Reply:
Wish I'd seen this before I spent the time debugging this LOL

Reply:
I had thought about pinging you on discord about it, now I wish I had. Oh well